src/DGtal/shapes/MeshHelpers.h

```diff
@@ -0,0 +1,149 @@
+ * Laboratory of Mathematics (CNRS, UMR 5127), University of Savoie, France
+ *
+ * @date 2017/02/11
+ *
+ * Header file for module MeshHelpers.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(MeshHelpers_RECURSES)
+#error Recursive header files inclusion detected in MeshHelpers.h
+#else // defined(MeshHelpers_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define MeshHelpers_RECURSES
+
+#if !defined MeshHelpers_h
+/** Prevents repeated inclusion of headers. */
+#define MeshHelpers_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include "DGtal/base/Common.h"
+#include "DGtal/topology/CCellEmbedder.h"
+#include "DGtal/topology/CDigitalSurfaceContainer.h"
+#include "DGtal/topology/DigitalSurface.h"
+#include "DGtal/shapes/TriangulatedSurface.h"
+#include "DGtal/shapes/Mesh.h"
+//////////////////////////////////////////////////////////////////////////////
+
+namespace DGtal
+{
+
+  /////////////////////////////////////////////////////////////////////////////
+  // template class MeshHelpers
+  /**
+   * Description of template class 'MeshHelpers' <p>
+   * \brief Aim: Static class that provides builder and converters between meshes.
+   */
+  class MeshHelpers
+  {
+    // ----------------------- Static services ------------------------------
+  public:
+    /// Builds a triangulated surface (class TriangulatedSurface) from
+    /// a mesh (class Mesh). Note that a triangulated surface contains
+    /// only triangles, so faces of the input mesh are (naively)
+    /// triangulated (triangles (0,1,2), (0,2,3), (0,3,4),
+    /// etc). Furthermore, the output triangulated surface rebuilds a
+    /// topology between faces.
+    ///
+    /// @tparam Point the type for points.
+    /// @param[in]  mesh the input mesh.
+    /// @param[out] trisurf the output triangulated surface mesh.
+    ///
+    /// @return 'true' on success, 'false' if the input \a mesh was
+    /// not a combinatorial surface.
+    template <typename Point>
+    static
+    bool mesh2TriangulatedSurface( const Mesh<Point>& mesh,
+                                   TriangulatedSurface<Point>& trisurf );
+
+    /// Builds a triangulated surface (class TriangulatedSurface) from
+    /// the dual graph of a digital surface (class
+    /// DigitalSurface).
+    ///
+    /// @note that a triangulated surface contains only triangles, so
+    /// faces of the input dual graph of the digital surface mesh are
+    /// triangulated by adding a new vertex at the barycenter of the
+    /// face vertices.
+    ///
+    /// @tparam DigitalSurfaceContainer the container chosen for the digital surface.
+    /// @tparam CellEmbedder the embedder chosen for the digital surface.
+    ///
+    /// @param[in]  dsurf the input digital surface.
+    /// @param[in]  cembedder the embedder for n-1-cells of the digital surface, which are vertices in the output triangulated surface.
```

Comment:
well.. we are dealing with 2-surfaces in dim 3, you can simply talk about 2-cells.